Makefile

```diff
@@ -1,5 +1,5 @@
 GO ?= go
-EPOCH_TEST_COMMIT ?= 78aae688e2932f0cfc2a23e28ad30b58c6b8577f
+EPOCH_TEST_COMMIT ?= 1cc5a27
```

Comment:
Just quadruple checking.  Is this value correct?  It seems shorter than I'd expect, especially in comparison to the previous value.